kube-deploy/group_vars/all.yml

```diff
@@ -18,6 +18,8 @@ bootstrap_os: ubuntu
 public_iface: enp0s3
 nat_iface: enp0s3
 proxy_enable: false
+#Only enable this if you are deploying on ARM or ARM64 with Fannel SDN provider
+arm_enabled: false
```

Comment:
@aric49 one approach I was thinking was to read off of the system for the type of architecture, rather than including additional flags for each of them. For instance, Ansible can be used to return system traits...use that rather than piling on our `global_vars` inputs.